packages/react-app/src/lib/bridge.js

```diff
@@ -87,20 +87,20 @@ export const fetchToToken = async fromToken => {
   };
 };
 
-export const fetchTokenLimits = async token => {
+export const fetchTokenLimits = async (token, walletProvider) => {
```

Comment:
does this work always ? what happens when user is not logged in ?

Reply:
this is after you've removed that check for ethersProvider. what if it is undefined? or is there no situation when it is undefined?

Reply:
fixed this [here](https://github.com/raid-guild/xdai-omnibridge/pull/105/files#diff-add24a0bb2ed915f99d188be48bd93466adc298bd04646fa14c8a1f8f62f54a6R76)